spec/valkyrie/sequel/persister_spec.rb

```diff
@@ -34,6 +34,62 @@ class TinyResource < Valkyrie::Resource
     end
   end
 
+  describe "save with optimistic locking being turned on later" do
+    it "doesn't break" do
+      class OptimisticResource < Valkyrie::Resource
+      end
+      resource = OptimisticResource.new
+      object = persister.save(resource: resource)
```

Comment:
I don't know if the token nil or zero at this point. You might add an expect statement to clarify that.